app/javascript/packs/OAS3Autocomplete.js

```diff
@@ -32,12 +32,17 @@ const addAutocompleteToParam = (param: Param, accountData: AccountData): Param =
     : param
 }
 
+const getPathParameters = (path) => {
```

Comment:
Are we sure `path` is always defined as well?

Reply:
yes, is the name of the path

Reply:
checking `ResponseBody` type, it seems it is, but not its contents